src/renderer/i18n/en/swap.ts

```diff
@@ -9,7 +9,9 @@ const swap: SwapMessages = {
   'swap.balance': 'Balance',
   'swap.output': 'Output',
   'swap.drag': 'Drag to swap',
-  'swap.searchAsset': 'Search Asset'
+  'swap.searchAsset': 'Search Asset',
+  'swap.errors.amount.balanceShouldCoverChainFee': 'Balance for {asset} should cover a fee',
+  'swap.errors.amount.outputShouldCoverChainFee': 'Output should cover a fee {fee}'
```

Comment:
Could you use [similar wording as we will have in Withdraw](https://github.com/thorchain/asgardex-electron/pull/662/files#diff-db167fd729cf28a10cb5727884e1a15f833e3e662f901f6f4e63aa691145b4f6R33-R36) _(you could not know it before...)_ It will help users to see different fees.